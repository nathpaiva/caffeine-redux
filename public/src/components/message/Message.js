import React from 'react';

const style = {
  error: {
    backgroundColor: '#982d2d',
    padding: '15px 0',
    textAlign: 'center',
    fontSize: '15px',
    color: '#fff',
    fontFamily: '"Roboto", sans-serif',
  },
  success: {
    backgroundColor: '#469a46',
    padding: '15px 0',
    textAlign: 'center',
    fontSize: '15px',
    color: '#fff',
    fontFamily: '"Roboto", sans-serif',
  }
}

const Input = ({ className, msg }) => {
  return (
    <div style={style[className]}>
      {msg}
    </div>
  )
};

export default Input;
