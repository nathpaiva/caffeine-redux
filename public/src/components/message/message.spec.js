'use stict';

import React from 'react';
import renderer from 'react-test-renderer';

import Message from './Message';


describe('#Test Message Component', () => {
  it('#Should Message with success', () => {
    const tree = renderer.create(<Message className='success' msg='Success message' />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('#Should Message with error', () => {
    const tree = renderer.create(<Message className='error' msg='Error message' />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
