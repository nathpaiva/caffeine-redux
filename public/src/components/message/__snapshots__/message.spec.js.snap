// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Test Message Component #Should Message with error 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#982d2d",
      "color": "#fff",
      "fontFamily": "\\"Roboto\\", sans-serif",
      "fontSize": "15px",
      "padding": "15px 0",
      "textAlign": "center",
    }
  }
>
  Error message
</div>
`;

exports[`#Test Message Component #Should Message with success 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#469a46",
      "color": "#fff",
      "fontFamily": "\\"Roboto\\", sans-serif",
      "fontSize": "15px",
      "padding": "15px 0",
      "textAlign": "center",
    }
  }
>
  Success message
</div>
`;
